include/utils/ExtrusionLine.h

```diff
@@ -228,6 +228,47 @@ struct ExtrusionLine
 
         return ret;
     }
+
+    /*!
+     * Create a true-extrusion area shape for the path; this means that each junction follows the bead-width
+     * set for that junction.
+     */
+    Polygons toExtrusionPolygons() const
```

Comment:
maybe we should add [[maybe_unused]] for this function?

Reply:
https://github.com/Ultimaker/CuraEngine/pull/2003/commits/f2930878300de0baf7b115fc32f65e597292856e